docs/spec/v1beta2/imagepolicies.md

```diff
@@ -250,6 +252,19 @@ spec:
 In the above example, the timestamp value from the tag pattern is extracted and
 used in the policy rule to determine the latest tag.
 
+### Digest Reflection
+
+`.spec.digestReflectionPolicy` is an optional field that governs the reflection of the selected image's
+digest in the ImagePolicy's `.status.latestDigest` field. The field has three possible values:
+
+- `null`: If the field is set to `null` (or not set at all) the digest will not be reflected at all.
```

Comment:
Can we introduce `Never`, it would be the equivalent to omitting the field from spec. Some people like to explicitly disabled features and using `null` can be problematic for SSA.